lib/typhoeus/version.rb

```diff
@@ -1,5 +1,5 @@
 module Typhoeus
 
   # The current Typhoeus version.
-  VERSION = '1.3.1'
+  VERSION = '1.4.1'
```

Comment:
What happened to `1.4.0`?

Reply:
lol sorry not sure what happened to my brain there

Reply:
Happens to the best of us. Thank you for taking this one over! ❤️